packages/nitro-protocol/jest.config.js

```diff
@@ -1,7 +1,11 @@
 module.exports = {
   setupFilesAfterEnv: ['./jest.setup.js'],
   collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}'],
-  reporters: ['default', '@statechannels/jest-gas-reporter'],
+  // TODO: It looks like jest is having some problems resolving to the jest-gas-reporter in node_modules
+  // We need to figure out what the exact issue is and a way to work around it
+  // reporters: ['default', '@statechannels/jest-gas-reporter'],
```

Comment:
I believe you saw this issue because you ran `yarn test` before running `yarn build`. You need to run `tsc -b` once in the `nitro-protocol` package such that the [references](https://github.com/statechannels/monorepo/blob/master/packages/nitro-protocol/tsconfig.json#L19) (see [here](https://www.typescriptlang.org/docs/handbook/project-references.html)) built `@statechannels/jest-gas-reporter`.

Reply:
I think we can change `main` to export a `.ts` file: https://github.com/statechannels/monorepo/blob/master/packages/jest-gas-reporter/package.json#L6

Reply:
Wouldn't `lerna` have run `yarn build` before `yarn test`? I was seeing this on circleCI (see https://circleci.com/gh/statechannels/monorepo/69)

Reply:
This was fixed by changing `jest-gas-reporter` to export a `.ts`

Reply:
I tried this but I'm still seeing the error so I've reverted to the commented out version for now.

Reply:
Here's an example of the failure https://circleci.com/gh/statechannels/monorepo/104

Reply:
Ah I figured it out ... because I added `composite: true` to `tsconfig.json` of `@statechannels/jest-gas-reporter` the output directory structure was `lib/src/*` not `lib/*`. Fixed in 1c446b16.